zc_common/jwt_auth/tests.py

```diff
@@ -50,3 +53,38 @@ def get_service_endpoint_urls(urlconfig=None, default_value='1'):
         result_urls.append(url)
 
     return result_urls
+
+
+class AuthenticationMixin:
+    def create_user(self, roles, user_id):
+        return User(roles=roles, pk=user_id)
+
+    def create_user_token(self, user):
+        payload = jwt_payload_handler(user)
+        token = "JWT {}".format(jwt_encode_handler(payload))
+        return token
+
+    def create_service_token(self, service_name):
+        payload = service_jwt_payload_handler(service_name)
+        token = "JWT {}".format(jwt_encode_handler(payload))
+        return token
+
+    def get_staff_token(self, user_id=1):
+        staff_user = self.create_user(['user', 'staff'], user_id)
+        return self.create_user_token(staff_user)
+
+    def get_user_token(self, user_id=1):
+        user = self.create_user(['user'], user_id)
+        return self.create_user_token(user)
+
+    def get_guest_token(self, user_id=1):
+        user = self.create_user(['user'], user_id)
+        setattr(user, 'isGuest', True)
```

Comment:
I think there was a misunderstanding due to overloading of the term 'guest'. Guest users having role of 'user', but Anonymous users having the role of 'guest' is really confusing.